content/intro-to-storybook/react/it/simple-component.md

```diff
@@ -298,31 +298,31 @@ export default function Task({ task: { id, title, state }, onArchiveTask, onPinT
 + };
 ```
 
-Now a warning in development will appear if the Task component is misused.
+Ora apparirà un avviso in fase di sviluppo se il componente Task viene utilizzato in modo errato.
 
 <div class="aside">
-💡 An alternative way to achieve the same purpose is to use a JavaScript type system like TypeScript to create a type for the component properties.
+💡 Un modo alternativo per raggiungere lo stesso obiettivo è utilizzare un sistema di tipi JavaScript come TypeScript per creare un tipo per le proprietà del componente.
 </div>
 
-## Component built!
+## Componente Costruito!
 
-We’ve now successfully built out a component without needing a server or running the entire frontend application. The next step is to build out the remaining Taskbox components one by one in a similar fashion.
+Abbiamo ora costruito con successo un componente senza aver bisogno di un server o di eseguire l'intera applicazione frontend. Il passo successivo è costruire uno per uno i restanti componenti di Taskbox in modo simile.
 
-As you can see, getting started building components in isolation is easy and fast. We can expect to produce a higher-quality UI with fewer bugs and more polish because it’s possible to dig in and test every possible state.
+Come puoi vedere, iniziare a costruire componenti in isolamento è facile e veloce. Possiamo aspettarci di produrre un'interfaccia utente di maggiore qualità con meno bug e più rifinita perché è possibile scavare e testare ogni stato possibile.
```

Comment:
I suggest to change **scavare** with **approfondire**. I know that literally "dig in" means "scavare" but in this case is used in slang.